### ISSUE TITLE
about the T2I results 

### ISSUE THREAD

**Issue by deropty** [YangLing0818/IterComp#6]
Thank you for you wonderful work. However, I can not reproduce the results in paper using the provided model in https://huggingface.co/comin/IterComp. For example, the color metric is 0.70, which is much less than the result (0.79) in paper.

**Comment by Cominclip** [comment]
Hello! Thank you for your interest in our project. Currently, we have only uploaded a preliminary model on Huggingface. Once our paper is finalized, we will release a more powerful model that corresponds to the metrics reported in the paper, along with our composition-aware model preference dataset and three pre-trained reward models. Additionally, please note that the results can be influenced by various factors such as seed, negative prompt, and scheduler settings.
